### ISSUE TITLE
Are tags shared among notebooks?

### ISSUE THREAD

**Issue by f0nzie** [tamlok/vnote#1198]
OS Version (操作系统): Linux
VNote Version (Help - About VNote - Version: )(VNote版本): 2.8
Symptoms (表现): tags are common only to one particular notebook
How to Repro (如何复现): create independent notebooks, add tags to notes, tags not shared

I am heavily experimenting with VNote and created ten different notebooks, by different activities or disciplines I am working on. Now that I have considerable amount of notes I found out that `tags` are shared ONLY within the notebooks for the notes that reside there and for which they were created. Is that the normal behavior? Is that feature or a bug?

Or, in other words, can the tags be made global, in such a way they are shared across all the notebooks?

Thanks.

**Comment by tamlok** [comment]
No. Each notebook has its own tags. But you could search tags among several notebooks.
